.github/FUNDING.yml

```diff
@@ -0,0 +1 @@
+custom: https://pandas.pydata.org/donate.html
```

Comment:
does this need a newline?

Reply:
Looks like other projects have a newline in their yml, will add one.

Reply:
lol thanks